spec/support/api_spec_helper.rb

```diff
@@ -114,7 +114,7 @@ def basic_authorize(user, password)
   end
 
   def update_user_role(role, *identifiers)
-    return if identifiers.blank?
+    return if identifiers.compact.blank?
```

Comment:
This change fixes a bug that @abellotti and I found while adding tests. It handles the case where there is no user associated with a request.